src/log_outgoing_requests/models.py

```diff
@@ -1,41 +1,56 @@
+from urllib.parse import urlparse
+
 from django.db import models
+from django.utils.functional import cached_property
 from django.utils.translation import gettext_lazy as _
 
 
 class OutgoingRequestsLog(models.Model):
+    url = models.URLField(
+        verbose_name=_("URL"),
+        blank=True,
+        default="",
+        help_text=_("The url of the outgoing request."),
+    )
     hostname = models.CharField(
```

Comment:
I'd put a comment here to note that `.hostname` is obviously part of `.url` but we also have it as a field so we can filter on it in the admin.